.github/workflows/docker-build-static.yml

```diff
@@ -484,7 +498,10 @@ jobs:
           push: ${{ github.event_name != 'pull_request' }}
           platforms: linux/amd64,linux/arm64
           file: ./.github/workflows/Dockerfile.ci.alpine
-          tags: nitnelave/lldap:latest, nitnelave/lldap:latest-alpine
+          tags: |
+            nitnelave/lldap:latest, nitnelave/lldap:latest-alpine
```

Comment:
I configured an "lldap" organization on dockerhub as well, can we add lldap/lldap here and below?

Reply:
done!